Scripts/RemediationScripts/AutoRemediation.ps1

```diff
@@ -168,7 +168,9 @@ Enter the choice (1|2)";
                 ($control.ControlId -eq "Azure_FrontDoor_CDNProfile_DP_Use_Secure_TLS_Version_Trial") -or
                 ($control.ControlId -eq "Azure_SQLServer_AuthN_Dont_Allow_Public_Network_Access") -or
                 ($control.ControlId -eq "Azure_RedisCache_DP_Use_Secure_TLS_Version") -or
-                ($control.ControlId -eq "Azure_Bastion_AuthZ_Disable_Shareable_Link")){
+                ($control.ControlId -eq "Azure_Bastion_AuthZ_Disable_Shareable_Link") -or
```

Comment:
Just to confirm, only 2 out of 3 controls are being covered in AutoRemediation right?

Reply:
Yes Aakash. BRS is feasible for only two controls and both are being covered in AutoRemediation.